content/events/2024-in-person-conference/_index.md

```diff
@@ -5,50 +5,82 @@ description = "Nordic RSE – Research Software Engineers in the Nordics – Con
 
 # Nordic-RSE in person conference 2024 <span style="color: gray;">#NordicRSEconf</span>
 
+
 **May 30-31**
 
 Registration will open in January 2024
 
 
+## What is it?
+
+This is the first Nordic RSE in person conference. We are an association of researchers who use software and people who write
+that software. So this is not like most academic conferences in that the focus is not on the results of research, but rather
+on the tools that underlie those results.
+
+Most research is already built on a foundation of sofware tools and the quality and reliability of those tools is key to
+producing reliable results. And the importance of software keeps growing. In this conference we want to talk about what software
```

Comment:
I would say "In this conference we want to talk about what pipelines you develop and use for your research, and how you use it, the software you develop, and how you got about developing it".

The reason is that for some of us it might be not only software, but also hardware that they develop.